src/main/java/racing/CarRacingGameMachine.java

```diff
@@ -0,0 +1,21 @@
+package racing;
+
+import racing.inout.InPutView;
+import racing.inout.OutPutView;
+
+public class CarRacingGameMachine {
+
+    public static void main(String[] args) {
+
+        String names = InPutView.name();
+        int howManyTimesToTry = InPutView.howManyTimesToTry();
+
+        CarFactory carFactory = new CarFactory();
+        ParticipationCars participationCars = carFactory.settingRacingCars(names);
```

Comment:
```suggestion
        ParticipationCars participationCars = ParticipationCars.of(names);
```

저는 정적 메소드 팩토리 패턴을 사용해서 
이렇게 해볼것 같네요~ 😄